easybuild/easyconfigs/c/CP2K/CP2K-9.1-foss-2022a.eb

```diff
@@ -0,0 +1,39 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'CP2K'
+version = '9.1'
+
+homepage = 'https://www.cp2k.org/'
+description = """CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular
+ simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different
+ methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and
+ classical pair and many-body potentials. """
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+source_urls = ['https://github.com/cp2k/cp2k/releases/download/v%(version)s.0/']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['fedb4c684a98ad857cd49b69a3ae51a73f85a9c36e9cb63e3b02320c74454ce6']
+
+dependencies = [
+    ('Libint', '2.6.0', '-lmax-6-cp2k'),
```

Comment:
@alinelena Is there a strict need for Libint 2.6.0 here?
See also #16073 ...

Reply:
nope if 2.7.1 or whatever is available shall be used instead.

Reply:
fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16007/commits/8d95a7d96fd1ed37c1752c5830ee327c1a9ed95d